AzureCalling/Controllers/CallViewController.swift

```diff
@@ -236,7 +236,22 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
     @IBAction func onToggleMute(_ sender: UIButton) {
         sender.isSelected = !sender.isSelected
-        sender.isSelected ? callingContext.mute { _ in} : callingContext.unmute { _ in}
+        sender.isSelected ? callingContext.mute(completionHandler: updateMuteIndicator) : callingContext.unmute(completionHandler: updateMuteIndicator)
+    }
+
+    private func updateMuteIndicator(_ result: Result<Void, Error>) {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
Perhaps renaming into `updateLocalMuteIndicator(isMuted: Bool)`. the parameter `Result` doesn't seem necessary
You can rewrite the `onToggleMute` function to make it more readable

Reply:
And in 1:1 group call, do we need to update local mute indicator?

Reply:
Will work on this after the main branch has been updated to use CallingSDK 12 since it relates to `call.isMuted`

Good point on the 1:1 group call though, seems that we can't see our own label. There's no need to update the indicator if it's 1:1

Reply:
Removed `updateLocalMuteIndicator(isMuted: Bool)` altogether after refactoring to use `call.isMuted`.

About 1:1 group call, tried to not update local mute indicator, but doesn't work as expected since any changes to the participant views would result in outdated local participant label. We still need to update it in 1:1 group call.